dev/com.ibm.websphere.appserver.features/visibility/public/mpTelemetry-1.0/io.openliberty.mpTelemetry-1.0.feature

```diff
@@ -12,12 +12,12 @@ IBM-API-Package: \
   io.opentelemetry.api;type="stable",\
   io.opentelemetry.api.trace;type="stable"
 -features=\
-  io.openliberty.jakarta.annotation-2.0, \
-  io.openliberty.restfulWS-3.0, \
+  io.openliberty.jakarta.annotation-2.1; ibm.tolerates:="2.0", \
+  io.openliberty.restfulWS-3.1; ibm.tolerates:="3.0", \
   io.openliberty.mpRestClient-3.0,\
```

Comment:
Will need to have jsonp dependency with tolerates if it is going to work with EE10 and 9 since mpRestClient will have the tolerates.